### PR TITLE
Retrying validation issue

### PR DESCRIPTION
**Pull request by prabh-t**

### CODE REVIEW

File: src/utils/ingest-client/ingest-file-validator.ts [hunk 1]
```diff
@@ -30,22 +31,39 @@ class IngestFileValidator {
...
+            } else {
+                const imageUrl = validationImage.imageUrl;
+                return IngestFileValidator._validateFile(fileName, uploadAreaUuid, imageUrl, this.uploadClient).then(validationJobId => {
+                    return Promise.resolve({
                             validationId: validationJobId,
                             checksums: fileChecksums,
-                            jobCompleted: false
-                        });
+                            jobCompleted: false,
+                            validationJob: null // reset validationJob
```
Comment (rolando-ebi) [inline src/utils/ingest-client/ingest-file-validator.ts]: Should this be `validationReport: null`?

Comment (prabh-t) [reply src/utils/ingest-client/ingest-file-validator.ts]: You're right. I don't think it's actually needed to be set anyway.

Comment (prabh-t) [reply src/utils/ingest-client/ingest-file-validator.ts]: OK, it's actually needed to reset any previous job report. Updated now.

File: src/utils/ingest-client/ingest-file-validator.ts [hunk 2]
```diff
@@ -30,22 +31,39 @@ class IngestFileValidator {
...
+            const completed = validationJob.jobCompleted;
+            if (!completed) {
+                return Promise.reject(new FileCurrentlyValidatingError());
+            }
+
+            if (validationJob.validationReport) {
+                const fileSha1 = fileChecksums.sha1;
+                const validatedSha1 = validationJob.checksums.sha1;
+
+                if (validationJob.validationReport.validationState == 'VALID' && fileSha1 == validatedSha1)
```
Comment (prabh-t) [inline src/utils/ingest-client/ingest-file-validator.ts]: we can be a bit more flexible here with the condition that triggers revalidation.

**Pull request merged**